### DISCUSSION TITLE
Preferred Branch for New Deployment

### DISCUSSION THREAD

**Discussion by monojit18** [Sunbird-Ed/Community#191] [Installation] [answered]
Dear Sunbird DevOps team,

For starting a new instance of Sunbird on any platform, which branch we should be using? Is it 5.0.0?
And if so then what should be the corresponding Build and Deploy tags for each pipeline job - should we still follow the list: http://docs.sunbird.org/latest/developer-docs/server-installation/build-and-deploy/ or a different one?

Can you please help us with this info so that we can move forward?

**Comment by beepdot** [comment]
@santhosh-tg @saiakhil46 Can you please suggest on this?

**Reply by monojit18** [reply]
@santhosh-tg, @saiakhil46 Can you please share your suggestion here?
Thanks in advance.

**Reply by santhosh-tg** [reply, marked as answer]
@monojit18  Please use release-4.10.0 branch  and refer this doc for Build and Deploy tags

https://ed.sunbird.org/use/prerequisites-for-your-own-sunbird-ed-instance/platform/build  (The documentation will be eventually moved from docs.sunbird.org to here )
